tests/foreman/api/test_email.py

```diff
@@ -0,0 +1,32 @@
+"""API Tests for the email notification feature"""
+
+import sys
+
+if sys.hexversion >= 0x2070000:
+    import unittest
+else:
+    import unittest2 as unittest
+
+from robottelo.common.constants import NOT_IMPLEMENTED
+from robottelo.test import APITestCase
+
+
+class EmailTestCase(APITestCase):
+    """API Tests for the email notification feature"""
+
+    @unittest.skip(NOT_IMPLEMENTED)
```

Comment:
`robottelo.common.decorators.stubbed` can also be used. See documentation [here](http://robottelo.readthedocs.org/en/latest/api/robottelo.common.html#robottelo.common.decorators.stubbed) and an example of a module which uses it [here](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/api/test_hostgroup.py).